packages/ember-glimmer/lib/ember-metal-views/index.js

```diff
@@ -13,6 +15,38 @@ class DynamicScope {
   }
 }
 
+const RENDERED_VIEWS = [];
+// let LAST_TAG_VALUE;
+
+function maybeRerender() {
+  // if (CURRENT_TAG.validate(LAST_TAG_VALUE)) { return; }
+  for (let i = 0; i < RENDERED_VIEWS.length; i += 2) {
+    let view = RENDERED_VIEWS[i];
+    let renderer = RENDERED_VIEWS[i + 1];
+    if (!view.isDestroyed) { renderer.rerender(view); }
+  }
+  // LAST_TAG_VALUE = CURRENT_TAG.value();
+}
+
+function scheduleRerender() {
```

Comment:
rerender and renderer  visually looks similar, maybe scheduleMaybeUpdate and maybeUpdate?